### ISSUE TITLE
New nuget packing does not preserve the dev / pre-release version (or suffix)

### ISSUE THREAD

**Issue by juanfranblanco** [Nethereum/Nethereum#117]
For reference:

https://github.com/NuGet/Home/issues/4337

This is affecting all the single libraries

**Comment by juanfranblanco** [comment]
Workaround:

```
SET releaseSuffix=rc2-fix
cd Nethereum.Hex
dotnet restore /property:ReleaseSuffix=%releaseSuffix%
dotnet pack 
```

Restore before packing passing the ReleaseSuffix and on Version.props
```
 <RelSuffix>$(ReleaseSuffix)</RelSuffix>
  <RelSuffix Condition=" '$(ReleaseSuffix)' == '' ">rc2-fix</RelSuffix>
 ```

**Comment by juanfranblanco** [comment]
Keeping it open to see the resolution of Nuget

**Comment by juanfranblanco** [comment]
This is fixed now in 15.6 Visual Studio https://github.com/NuGet/Home/issues/4337. Not a priority.

**Comment by juanfranblanco** [comment]
Keeping it as it is not a priority.
